pages/common/mutool.md

```diff
@@ -0,0 +1,23 @@
+# mutool
+
+> Convert PDF files, query information and extract data.
+
+- Convert pages 1-10 into 10 PNG images:
+
+`mutool convert -o {{image%d.png}} {{file.pdf}} {{1-10}}`
+
+- Convert pages 2, 3 and 5 of a PDF into text in the standard output:
+
+`mutool draw -F {{txt}} {{file.pdf}} {{2,3,5}}`
+
+- Concatenate all pages skipping the first from two PDFs into a new one:
```

Comment:
The wording here seems a bit confusing to me. Perhaps something like this would read better?:

```markdown
- Merge 2 PDFs by concatenation, skipping the first page of each:
```

What do you think?